src/Microsoft.ML.Tokenizers/Model/EnglishRoberta.cs

```diff
@@ -397,12 +423,13 @@ private IReadOnlyList<Token> ModifyTokenListOffsets(IReadOnlyList<Token> tokens,
         private static HighestOccurrenceMapping GetHighestOccurrenceMapping(Stream highestOccurrenceMappingStream) =>
             HighestOccurrenceMapping.Load(highestOccurrenceMappingStream);
 
-        private Dictionary<string, int> GetVocabulary(Stream vocabularyStream)
+        private Dictionary<StringSpanOrdinalKey, int> GetVocabulary(Stream vocabularyStream)
         {
-            Dictionary<string, int>? vocab;
+            Dictionary<StringSpanOrdinalKey, int>? vocab;
             try
             {
-                vocab = JsonSerializer.Deserialize<Dictionary<string, int>>(vocabularyStream) as Dictionary<string, int>;
+                JsonSerializerOptions options = new() { Converters = { new StringSpanOrdinalKeyConverter() } };
```

Comment:
This has now shown up a couple of times. Better to just put it into a static readonly singleton somewhere.